seed/functional/tests/test_browser.py

```diff
@@ -109,7 +104,7 @@ def test_accounts_page(self):
                     user=self.user, org=other_org, role='member'
                 )
 
-                page = mainpage(self, use_url=true)
```

Comment:
@tallus -- This is failing flake8 as being undefined. Can you look at this? I don't want to merge this to develop until this is fixed. Thanks!

```
/Users/nlong/working/seed/seed/functional/tests/test_browser.py:107:24: F821 undefined name 'mainpage'
/Users/nlong/working/seed/seed/functional/tests/test_browser.py:110:28: F821 undefined name 'accountspage'
/Users/nlong/working/seed/seed/functional/tests/test_browser.py:185:24: F821 undefined name 'mainpage'
/Users/nlong/working/seed/seed/functional/tests/test_browser.py:185:47: F821 undefined name 'true'
/Users/nlong/working/seed/seed/functional/tests/test_browser.py:187:28: F821 undefined name 'datasetslist'
/Users/nlong/working/seed/seed/functional/tests/test_browser.py:203:28: F821 undefined name 'datasetslist'
/Users/nlong/working/seed/seed/functional/tests/test_browser.py:204:41: F821 undefined name 'true'
/Users/nlong/working/seed/seed/functional/tests/test_browser.py:212:27: F821 undefined name 'datasetinfo'
/Users/nlong/working/seed/seed/functional/tests/test_browser.py:229:41: F821 undefined name 'row_delimiter'
/Users/nlong/working/seed/seed/functional/tests/test_browser.py:232:51: F821 undefined name 'row_delimiter'
/Users/nlong/working/seed/seed/functional/tests/test_browser.py:237:27: F821 undefined name 'datasetinfo'
/Users/nlong/working/seed/seed/functional/tests/test_browser.py:245:32: F821 undefined name 'datamapping'


```